extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/CacheAnnotationsTransformer.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.cache.deployment;
+
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.AUGMENTED_CACHE_INVALIDATE;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.AUGMENTED_CACHE_INVALIDATE_ALL;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.AUGMENTED_CACHE_LOAD;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.AUGMENTED_CACHE_STORE;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.CACHE_INVALIDATE;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.CACHE_INVALIDATE_ALL;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.CACHE_LOAD;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.CACHE_NAME_PARAMETER_NAME;
+import static io.quarkus.cache.deployment.CacheDeploymentConstants.CACHE_STORE;
+
+import org.jboss.jandex.AnnotationTarget.Kind;
+import org.jboss.jandex.AnnotationValue;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.MethodInfo;
+
+import io.quarkus.arc.processor.AnnotationsTransformer;
+
+/**
+ * @author Gwenneg Lepage
```

Comment:
We agreed on not using the `@author` tag and rely on the Git history for authorship.